poker-hand-evaluator/src/Main.java

```diff
@@ -1,58 +1,60 @@
 import evaluator.*;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
 public class Main {
     static DeckOfCards deck = new DeckOfCards();
     static List<Hand> winnerHand = new ArrayList<>();
 
     public static void main(String[] args) throws InvalidHandSizeException {
 
-        Player player1 = new Player("Player_A");
-        Player player2 = new Player("Player_B");
-
-        startOfDealing(player1);
-        startOfDealing(player2);
-
-        flopOfDealing(player1);
-        flopOfDealing(player2);
+        List<Player> playerList = new ArrayList<>();
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("How many players will play the Texas poker:");
+        int counterOfPlayer = scanner.nextInt();
+        scanner.nextLine();
+        for (int i = 0; i < counterOfPlayer; i++) {
+            String playerName = scanner.nextLine();
+            Player player = new Player(playerName);
+            playerList.add(player);
+        }
 
-        turnOfDealing(player1);
-        turnOfDealing(player2);
+        for (Player player : playerList) {
+            startOfDealing(player);
+        }
 
-        riverOfDealing(player1);
-        riverOfDealing(player2);
+        flopOfDealing();
+        turnOfDealing();
+        riverOfDealing();
```

Comment:
All this logic screams for some kind of a dealer class - this logic shouldn't be in your `Main` class